### ISSUE TITLE
Ignore \ensuremath in custom macros

### ISSUE THREAD

**Issue by 4e554c4c** [varkor/quiver#232]
\ensuremath should be ignored when in math mode

**Comment by varkor** [comment]
Unfortunately, KaTeX doesn't yet support this (https://github.com/KaTeX/KaTeX/issues/3687), so this is blocked on KaTeX adding the feature.

**Comment by varkor** [comment]
Closing in favour of https://github.com/varkor/quiver/issues/241.
